charts-packages/ag-charts-community/src/axis.ts

```diff
@@ -657,30 +686,69 @@ export class Axis<S extends Scale<D, number>, D = any> {
             gridlineGroupSelection = updateGridlines.merge(enterGridline);
         }
 
+        return gridlineGroupSelection;
+    }
+
+    private updateSelections({
+        tickCountOffset,
+        halfBandwidth,
+        gridLength,
+        ticks,
+    }: {
+        tickCountOffset: number;
+        halfBandwidth: number;
+        gridLength: number;
+        ticks?: any[];
+    }) {
+        const { scale } = this;
+        ticks = ticks ?? this.getTicks(tickCountOffset);
+        const data = ticks.map((t) => ({ tick: t, translationY: Math.round(scale.convert(t) + halfBandwidth) }));
+        const gridlineGroupSelection = this.updateGridLineGroupSelection({ gridLength, data });
+        const tickGroupSelection = this.updateTickGroupSelection({ data });
+
+        const translationYFn = (_: unknown, datum: any) => datum.translationY;
+        gridlineGroupSelection.attrFn('translationY', translationYFn);
+        tickGroupSelection.attrFn('translationY', translationYFn);
```

Comment:
It might be simpler to inline this, as you can then lose the irrelevant types?
```suggestion
        gridlineGroupSelection.attrFn('translationY', (_, datum) => datum.translationY);
        tickGroupSelection.attrFn('translationY', (_, datum) => datum.translationY);
```

Reply:
sure